sdk/core/Azure.Core/src/Pipeline/ResponseClassifier.cs

```diff
@@ -32,5 +33,39 @@ public virtual bool IsErrorResponse(Response response)
             var statusKind = response.Status / 100;
             return statusKind == 4 || statusKind == 5;
         }
+
+        public static bool IsTextResponse(Response response, out Encoding encoding)
```

Comment:
I was wondering if this should be virtual but it makes creating the exception slightly more verbose because it becomes a required parameter.

Reply:
I'll keep it static for now because with virtual error creation gets way too complicated and see how it plays out.